lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1970,5 +1970,41 @@ sub status_info {
     return $info;
 }
 
+# a condensed mapping when less states need to be differentiated, e.g. for
+# favicons based on state
+sub simplified_state {
+    my ($self) = @_;
+    my %simplified_state = (
+        SCHEDULED() => SCHEDULED,
```

Comment:
I would prefer a `,` instead of `()` to preserve the bareword here.

Reply:
alright. Actually I was confused by the bareword/constant confusion a lot, not that "SCHEDULED" would be seen as string with the big comma but that "TIMEOUT_EXCEEDED" was never exported from lib/OpenQA/Jobs/Constants.pm

Reply:
I think the fat comma is actually better here, since we are dealing with key/value pairs. Even if the constants do look a bit ugly this way.